package.json

```diff
@@ -1,20 +1,22 @@
 {
   "name": "rails_api_base",
   "private": "true",
-  "engines" : {
-    "node" : "20.10.0",
-    "yarn" : "1.22"
+  "engines": {
+    "node": "20.10.0",
+    "yarn": "1.22"
   },
   "dependencies": {
     "@activeadmin/activeadmin": "3.2.3",
     "@fortawesome/fontawesome-free": "6.6.0",
     "arctic_admin": "4.3.1",
     "esbuild": "0.23.0",
     "jquery": "3.7.1",
-    "jquery-ui": "1.14.0",
-    "jquery-ujs": "1.2.3"
+    "jquery-ui": "1.13.3",
+    "jquery-ujs": "1.2.3",
+    "sass": "^1.77.8"
   },
   "scripts": {
-    "build": "node esbuild.config.mjs"
+    "build": "node esbuild.config.mjs",
+    "build:css": "sass ./app/assets/stylesheets:./app/assets/builds --no-source-map --load-path=node_modules"
```

Comment:
Shouldn't the `build` script run this as well? Maybe we can call this `prebuild` so it runs automatically?

Reply:
I think this was added by the cssbundling gem. I mean, the default behavior.

Reply:
Yeah, I'm just saying that would be nice to have something like
```JSON
scripts: {
  "build": "yarn build:js && yarn build:css",
  "build:js": "node esbuild.config.mjs",
  "build:css": "sass ./app/assets/stylesheets:./app/assets/builds --no-source-map --load-path=node_modules"
}
```

That way `build` runs both process. Or another option would be to rename the `build:css` script to `prebuild` so it's always executed when we call `build`. (I like the first option because it's more flexible tbh).

Anyway, if you want to keep it this way, we might need to add the `build:css` step to our `Dockerfile.dev`